### DISCUSSION TITLE
新版本怎么开启同一行隔壁按键纠错的

### DISCUSSION THREAD

**Discussion by Alwaysliumx** [fcitx5-android/fcitx5-android#505] [Q&A | 问答]
就是按baudu，可以预测出baidu（百度）的候选词

**Comment by rocka** [comment]
需要使用自带的拼音输入法，在拼音设置最下面，将“纠错布局”设置为 QWERTY

**Comment by Alwaysliumx** [comment]
感谢回复，软件真的很好用发自我的手机-------- 原始邮件 --------发件人： rocka ***@***.***>日期： 2024年5月17日周五 11:57收件人： fcitx5-android/fcitx5-android ***@***.***>抄送： Alwaysliumx ***@***.***>, Author ***@***.***>主    题： Re: [fcitx5-android/fcitx5-android] 新版本怎么开启同一行隔壁按键纠错的 (Discussion #505)
需要使用自带的拼音输入法，在拼音设置最下面，将“纠错布局”设置为 QWERTY
